c/cmake_modules/BuildUtils.cmake

```diff
@@ -467,15 +467,13 @@ function(ADD_BENCHMARK REL_BENCHMARK_NAME)
       target_link_libraries(${BENCHMARK_NAME} PRIVATE ${ADBC_BENCHMARK_LINK_LIBS})
     endif()
     add_dependencies(benchmark ${BENCHMARK_NAME})
-    set(NO_COLOR "--color_print=false")
```

Comment:
This option doesn't exist for benchmark at all; benchmark documents having color output by default, but I didn't see a command line switch to toggle that. Figured this can be safely removed

Reply:
https://google.github.io/benchmark/user_guide.html